Zend/zend_fibers.stub.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-/** @generate-class-entries */
+/**
+ * @generate-class-entries
+ * @strict-properties
+ */
```

Comment:
These should be two separate comments. One is a file comment, one is a class comment.